docs/_layouts/default.html

```diff
@@ -1,5 +1,15 @@
 <!DOCTYPE html>
 <html>
+  {% assign protocol_scheme = page.url | absolute_url | truncate: 5, "" %}
+  {%- capture site_url -%}
+    {%- if protocol_scheme == "https" -%}
+      <!-- This is a hack to force using HTTPS... -->
+      <!-- The URL of the docs should be changed in the repo settings instead -->
```

Comment:
I believe these two lines are still the problem. See my comment here - https://github.com/box/spout/issues/689#issuecomment-543119194.